### PR TITLE
completed CC33

### PR DESCRIPTION
**Pull request by Gr8-Dayne**

### CODE REVIEW

**Comment by Gr8-Dayne** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: challenges/src/test/java/hashtable/TreeIntersectionTest.java [hunk 1]
```diff
@@ -1,13 +1,66 @@
...
+        treeTwo.root.right.left = new Node<>(18);
+        treeTwo.root.right.right = new Node<>(15);
+    }
+
     @Test
-    public void treeIntersection() {
+    public void treeIntersectionTest() {
+        endResult = treeIntersection(treeOne, treeTwo);
+        System.out.println("-----End Result-----");
+        System.out.println(endResult);
```
Comment (JamesDansie) [inline challenges/src/test/java/hashtable/TreeIntersectionTest.java]: This will give better information if in your PainfulExp.java you add a toString method;

    @Override
    public String toString() {
        return "PainfulExp{" +
                "key='" + key + '\'' +
                ", value=" + value +
                '}';
    }

Then you can see the key/value data.
